modules/test/playwright/pages/object-web/list-type/ListTypeDefinitionsPage.ts

```diff
@@ -35,6 +37,24 @@ export class ListTypeDefinitionsPage {
 		await this.savePicklistButton.click();
 	}
 
+	async addPicklistItem(
+		picklistName: string,
+		picklistItem: string,
+		picklistKey: string
+	) {
+		await this.page.getByRole('link', {name: picklistName}).click();
+		await this.page.frameLocator('iframe').getByLabel('Add Item').click();
+		await this.page
+			.getByPlaceholder('Text to translate...')
+			.fill(picklistItem);
+		await this.page.locator('input[name="name"]').fill(picklistKey);
```

Comment:
Done

Reply:
Yes, but it's not working because the label is no bound to the field. Please, add an `id` attribute to this `Input` [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ListTypeDefinition/ListTypeEntriesModal.tsx#L241). we are going to have something like this:
```
<Input
	disabled={modalType === 'edit'}
	error={errors.name}
	id="listTypeEntriesModalKeyInputField"
	label={Liferay.Language.get('key')}
	name="name"
	onChange={({target}) => handleKeyChange(target.value)}
	required
	value={itemKey ?? ''}
/>
```
Please, include this change in a separate commit.

Reply:
Hi @cgmonte , the test was not working after the change I made? 

Sure, I'll do the latest request. Thanks.